containers/Containerfile

```diff
@@ -1,9 +1,12 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest as builder
-RUN dnf install -y make git unzip gcc
+RUN dnf install -y make git unzip gcc openssl-devel
 RUN git clone https://github.com/wg/wrk.git --depth=1
+RUN git clone https://github.com/giltene/wrk2.git --depth=1
 RUN cd wrk && make -j $(nproc)
+RUN cd wrk2 && make -j $(nproc)
```

Comment:
Just ranting here, given the GitHub runners we got, we will just "wait" 
forever 😭 to build wrk & wrk2

Reply:
Not sure I follow - can you explain please?

Reply:
> Not sure I follow - can you explain please?

GitHub (Free) runners, that we get today are so small that we are spending >45mins to build Luajit/openssl-* per architecture. If we add QEMU to the mix to build/compile non-x86 architectures it might take even more time, and we observed the job eventually timing out.

And with wrk2, we might spend more time building & compiling it.

Reply:
Will explore the possibility of having an AWS SPOT instance for this effort.